src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <section class="card__image">
+      </section>
+
+      <section class="card__name card__name--text_medium">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </section>
+
+      <section class="card__code card__code--text_small">
+        Product code : 195434
+      </section>
+
+      <section class="card__rating">
+        <div class="stars stars--active">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <span class="card__review card__review--text_small">
+          Reviews: 5
+        </span>
+      </section>
+
+      <section class="price">
+        <span class="price__text">Price:</span>
+
+        <span class="price__amount price__amount--text_large">
+          $2,199
+        </span>
+      </section>
+
+      <section class="buy">
+        <a
+          href="#"
+          data-qa = "hover"
+          class="buy__button"
+        >
+          Buy
+        </a>
+      </section>
```

Comment:
no need to give it section tag. You can make this BLOCK but you need to give it also an element class so that you can position it accordingly

Reply:
Fixed.